Russian translation

dragomano opened the pull request:
Signed-off-by: Bugo <bugo@dragomano.ru>

dragomano committed:
Russian translation

Signed-off-by: Bugo <bugo@dragomano.ru>

SychO9 commented:
Thank you Bugo, I added one more helptext in my last commit could you please include that as well ?
https://github.com/SychO9/smf-fa-board-icons/blob/6a771deddf9eecf2aa52befdb4355200f88f14c4/FA-BoardIcons.english.php#L39

dragomano committed:
Update

Signed-off-by: Bugo <bugo@dragomano.ru>

dragomano commented:
> Thank you Bugo, I added one more helptext in my last commit could you please include that as well ?
> [smf-fa-board-icons/FA-BoardIcons.english.php](https://github.com/SychO9/smf-fa-board-icons/blob/6a771deddf9eecf2aa52befdb4355200f88f14c4/FA-BoardIcons.english.php#L39)
> 
> Line 39 in [6a771de](/SychO9/smf-fa-board-icons/commit/6a771deddf9eecf2aa52befdb4355200f88f14c4)
> 
>  $helptxt['fabi_board_icon'] = 'You can force use a single icon and color on all boards from <a href="'.$scripturl.'?action=admin;area=modsettings;sa=general">this page</a>';

Yes, updated :)
